.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: CI Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy: 
+      matrix : 
+        os: [ubuntu-latest, windows-latest]
+        java: [8, 11]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
```

Comment:
Should this use JDK11 or 8? I thought 8 was the minimum?

Reply:
The name is incorrect, it should be "Set up JDK"

This step is using `${{ matrix.java }}` so it is testing against JDK 8 and 11.